g11n-ws/vip-manager-l10n/src/main/java/com/vmware/l10n/source/dao/impl/S3SourceDaoImpl.java

```diff
@@ -37,14 +51,12 @@
 	/**
 	 * the path of local resource file,can be configured in spring config file
 	 **/
-	@Value("${source.bundle.file.basepath}")
+	@Value("${source.bundle.file.basepath:.}")
```

Comment:
Don't need to set it to . because no current directory in S3. Empty string means root folder.

Reply:
this default is .